src/main/java/com/transportcompany/transportcompany/repositories/UserRepository.java

```diff
@@ -5,9 +5,13 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface UserRepository extends JpaRepository<User, Long> {
     static User save(UserDTO userToSave) {
```

Comment:
The repository layer should not contain static blocks.
To save a `User` entity you don't need to declare a method. JpaRepository extends CrudRepository which has save, find, and many other methods that can help with CRUD operations.

Reply:
Done